index.js

```diff
@@ -18,5 +19,17 @@ module.exports = (red, green, blue) => {
 		throw new TypeError('Expected three numbers below 256');
 	}
 
-	return ((blue | green << 8 | red << 16) | 1 << 24).toString(16).slice(1);
+	if (typeof alpha === 'number') {
+		if (alpha < 1 && alpha >= 0) {
+			alpha = Math.round(256 * alpha).toString(16);
```

Comment:
There's no need for a toString operation. Just shift the color bytes by another 8 and tack on the alpha byte at the end.